manager/src/main/java/io/gingersnapproject/RuleResource.java

```diff
@@ -6,20 +6,30 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 
+import io.gingersnapproject.metrics.CacheAccessRecord;
+import io.gingersnapproject.metrics.CacheManagerMetrics;
 import io.smallrye.mutiny.Multi;
 
 @Path("/rules")
 public class RuleResource {
 
    @Inject
    Caches maps;
+   @Inject
+   CacheManagerMetrics metrics;
 
    @GET
    @Path("/{rule}/{key}")
    @Produces(MediaType.APPLICATION_JSON)
    public String get(String rule, String key) {
-      byte[] value = maps.get(rule, key);
-      return value == null ? null : new String(value);
+      CacheAccessRecord<String> cacheAccessRecord = metrics.recordCacheAccess();
```

Comment:
We're going to have multiple endpoints, so I think it makes sense that we record these metrics inside the `Caches` abstraction so that they will be recorded centrally.

Reply:
That would be better. My new PR https://github.com/gingersnap-project/cache-manager/pull/14 also moves the lazy loading inside of Caches as well, granted it also converts to using Uni as well which will conflict with this also.